webpack.config.js

```diff
@@ -0,0 +1,15 @@
+var webpack = require ('webpack')
+
+module.exports = {
+    entry: './source/likely.js',
+    output: {
+        filename: 'likely.js',
+        library: 'likely',
```

Comment:
Probably we’d better change this to `ilyabirman-likely` to match the npm module name?